README.md

```diff
@@ -66,7 +66,7 @@ To learn how to use restic backup client with REST backend, please consult [rest
           --tls-key string       TLS key path
       -V, --version              output version and exit
 
-By default the server persists backup data in `/tmp/restic`.  To start the server with a custom persistence directory and with authentication disabled:
+By default the server persists backup data in the O.S. temporary directory (`/tmp/restic` on Linux/BSD and others, in `%TEMP%\\restic` in Windows, etc).  To start the server with a custom persistence directory and with authentication disabled:
```

Comment:
Please use OS as abbreviation for operating system.

The documentation of TempDir ( https://pkg.go.dev/os#TempDir) states that it first looks into `$TMPDIR` if that is set on Unix systems.